task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,10 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 3 == 0 && num % 5 == 0) return 'FizzBuzz';
+    else if (num % 5 == 0) return 'Buzz';
+    else if (num % 3 == 0) return 'Fizz';
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"